--- src/sqladmin_litestar_plugin/__init__.py
+++ src/sqladmin_litestar_plugin/__init__.py
@@ -72,7 +72,7 @@
             if value is not Empty
         }
         self.starlette_app = Starlette()
-        self.admin = sqladmin.Admin(app=self.starlette_app, **admin_kwargs)
+        self.admin = sqladmin.Admin(app=self.starlette_app, **admin_kwargs)  # type: ignore[arg-type]
         self.starlette_app.add_middleware(PathFixMiddleware, base_url=self.admin.base_url)
         # disables redirecting based on absence/presence of trailing slashes
         self.starlette_app.router.redirect_slashes = False
@@ -91,7 +91,7 @@
             Performs, and unwinds, the necessary scope modifications for the SQLAdmin app.
             """
             try:
-                await self.starlette_app(_prepare_scope(scope, mount_path), receive, send)
+                await self.starlette_app(_prepare_scope(scope, mount_path), receive, send)  # type: ignore[arg-type]
             except Exception:
                 logger.exception("Error raised from SQLAdmin app")
